src/reporter/index.js

```diff
@@ -46,31 +47,36 @@ export default class Reporter {
         return find(this.reportQueue, i => i.test === testRun.test);
     }
 
-    _shiftReportQueue () {
-        // NOTE: a completed report item is always at the front of the queue.
-        // This happens because the next test can't be completed until the
-        // previous one frees all browser connections.
-        // Therefore, tests always get completed sequentially.
-        var reportItem = this.reportQueue.shift();
+    _shiftReportQueue (reportItem) {
+        if (!reportItem.testRunInfo) {
+            reportItem.testRunInfo = Reporter._createTestRunInfo(reportItem);
 
-        if (!reportItem.errs.length && !reportItem.test.skip)
-            this.passed++;
+            if (!reportItem.errs.length && !reportItem.test.skip)
+                this.passed++;
+        }
 
-        this.plugin.reportTestDone(reportItem.test.name, Reporter._createTestRunInfo(reportItem));
+        var currentFixture = null;
```

Comment:
I think the `_shiftReportQueue` method should start from this line because shifting starts here. Move the code above out of this function to the `test-run-done` event handler